roles/candlepin/tasks/artemis.yml

```diff
@@ -5,7 +5,7 @@
 
 - name: Set candlepin_artemis_client_dn
   ansible.builtin.set_fact:
-    candlepin_artemis_client_dn: "{{ openssl_response.stdout | replace('subject=', '') }}"
+    candlepin_artemis_client_dn: "{{ openssl_response.stdout | replace('subject=', '') | replace(' ', '') | replace(',', ', ') }}"
```

Comment:
Otherwise you get

```
2024-09-30 07:31:55,292 [thread=Thread-1 (activemq-netty-threads)] [=, org=, csid=] WARN  org.apache.activemq.artemis.core.server - AMQ222216: Security problem while authenticating: AMQ229031: Unable to validate user from 127.0.0.1:43804. Username: null; SSL certificate subject DN: CN=quadlet.example.com, O=Foreman, ST=NC, C=US
2024-09-30 07:31:55,304 [thread=Thread-1 (activemq-netty-threads)] [=, org=, csid=] WARN  org.apache.activemq.artemis.core.protocol.stomp - AMQ332069: Sent ERROR frame to STOMP client 127.0.0.1:43804: Security Error occurred: User name [null] or password is invalid
```

Reply:
`replace(' ', '')` I am surprised at this one as it compresses names, e.g. "O=Red Hat" would turn into "O=RedHat" which would be incorrect I'd think? If I look at our normal deployments, I think subject should be the only problem.

```
CN=hostname, OU=PUPPET, O=FOREMAN, ST=North Carolina, C=US
```

Reply:
So the real problem I was solving is the `, ` vs `,` -- java really wants those spaces between the DN parts.

You are right the "kill all spaces" first is bad.